fastlane.js

```diff
@@ -2,6 +2,7 @@
 const githubUsername = process.env.HUBOT_GITHUB_USERNAME
 const githubToken = process.env.HUBOT_GITHUB_TOKEN
 const organisation = "owncloud"
+const projectName = "QA/CI/TestAutomation"
 const projectNumber = 386
```

Comment:
When project 386 is found, there could be a check that the project name matches the name here - that will also help catch unexpected problems with the "majic project number"

Reply:
It is possible to get the project number from the project name directly from the graphql. That would be another extra query. But I assumed this number would remain the same every time.

Reply:
I found the query where the title can be fetch directly in the same request. I will add the check

Reply:
Yes, I think continue to query the "magic number" because that should not change.
I was just thinking of putting and extra check for confidence - when the query result comes, I suppose that there is a data item with the project name in it. That could be checked, and a message emitted if the name is not the expected name.